database/metadata/databases/default/tables/public_crashes.yaml

```diff
@@ -129,6 +129,7 @@ select_permissions:
         - crash_timestamp
         - created_at
         - created_by
+        - diagram_zoom_rotate
```

Comment:
At one point, I believe you were talking about adding support to save the translation of the image as well (x,y offsets). Is that still in the cards? If so, maybe this field could be named `diagram_transform` or `diagram_transform_state`? Maybe even if it's not in the cards..

This isn't a big deal of course, I'm just trying to think of something that is perhaps more semantically accurate.